pitest-maven/src/main/java/org/pitest/maven/PitMojo.java

```diff
@@ -47,7 +47,7 @@
    * @parameter expression="${targetClasses}"
    *
    */
-  protected ArrayList<String>         targetClasses;
+  private List<String>         targetClasses;
```

Comment:
I'd assumed this just enforced the minimum version for building pitest. Does it set a minimum for someone running the built plugin?